### PR TITLE
Refactor docs to clarify open source and Enterprise

### PR DESCRIPTION
**Pull request by LappleApple**
**What changed?**

This is another increment of edits to the user docs, this time applying more context and working to tease apart OSS from Enterprise so that we can reorder pages in a simpler flow for each.

**Why was this change made?**

Currently there's friction in getting started.

### CODE REVIEW

File: website/docs/getting-started/ui.mdx [hunk 1]
```diff
@@ -1,112 +1,139 @@
...
 ---
 
-# Part 1: Explore the GitOps UI
+# Step 2: Explore the Weave GitOps Open Source UI
 
-Weave GitOps provides insights into your application deployments, and makes continuous delivery with GitOps easier to adopt and scale across your teams. We will now login to the dashboard web UI and start to explore the state of our GitOps deployments.
+The Weave GitOps user interface enables you to manage and view all of your applications in one place. This documentation gives you an overview of the Weave GitOps Open Source UI. 
+
+:::tip To check out Weave GitOps Enterprise's UI, which provides an even richer user experience, please contact sales@weave.works.
+:::
```
Comment (Himangini) [inline website/docs/getting-started/ui.mdx:11]:
```suggestion
:::tip 
To check out Weave GitOps Enterprise's UI, which provides an even richer user experience, please contact sales@weave.works.
:::
```

File: website/docs/getting-started/ui.mdx [hunk 2]
```diff
@@ -1,112 +1,139 @@
...
 
-This table view shows the reported status so you can understand whether a reconciliation has been successful, and when they have last been updated. You can also see where the Flux objects are deployed and which `Source` object they are reconciling from; clicking the name of the Source it will take you to a detail view for the given source object. The view automatically updates every few seconds so you know the current state of your system.
+The table view shows you the reported status so you can understand whether a reconciliation has been successful, and when it was last updated. You can also see where the Flux objects are deployed, and which `Source` object they are reconciling from. Clicking the name of the Source will take you to a detail view for the given Source object. The view automatically updates every few seconds so you know the current state of your system.
 
-You can search for and filter objects by `Name` by clicking the magnifying glass, or filter by `Type` by clicking the strawberry icon to its right.
+:::tip For more information about Sources, please take a look at the [Flux documentation](https://fluxcd.io/flux/concepts/#sources).
+:::
```
Comment (Himangini) [inline website/docs/getting-started/ui.mdx:54]:
```suggestion
:::tip 
For more information about Sources, please take a look at the [Flux documentation](https://fluxcd.io/flux/concepts/#sources).
:::
```

File: website/docs/installation/weave-gitops.mdx [hunk 3]
```diff
@@ -55,24 +61,27 @@ To upgrade to the latest version, run this command:
...
+We recommend upgrading the CLI before running bootstrap to upgrade the controllers with `flux bootstrap`.
+
 Find which version is installed with `flux -v`, and use that for `flux bootstrap --version=v<CLI-VERSION>`.
 
-With Bash, you can run `sudo curl -s https://fluxcd.io/install.sh | sudo FLUX_VERSION=<VERSION> bash`. 
+With Bash, you can run `sudo curl -s https://fluxcd.io/install.sh | sudo FLUX_VERSION=<VERSION> bash`.
 
-We recommend upgrading the CLI before running bootstrap to upgrade the controllers with `flux bootstrap`.
+:::tip If you want to install an older version of Flux CLI, you can download the binary for your OS from the [releases page](https://github.com/fluxcd/flux2/releases).
+:::
```
Comment (Himangini) [inline website/docs/installation/weave-gitops.mdx:71]:
```suggestion
:::tip 
If you want to install an older version of Flux CLI, you can download the binary for your OS from the [releases page](https://github.com/fluxcd/flux2/releases).
:::
```

File: website/docs/installation/weave-gitops.mdx [hunk 4]
```diff
@@ -168,24 +172,26 @@ In this section we will do the following:
...
 
 	:::warning
-	This command stores a hash of a password.  This is relatively safe for demo
+   This command stores a hash of a password. This is relatively safe for demo
 	and testing purposes, but we strongly recommend using a more secure method of storing secrets
 	(such as [Flux's SOPS integration][sops]) for production systems.
 
 	Our docs on [securing access to the dashboard][dash-access] provide additional guidance and alternative login methods.
+
 	:::
```
Comment (Himangini) [inline website/docs/installation/weave-gitops.mdx:181]: spacing changes here can be omitted

File: website/docs/intro.mdx [hunk 5]
```diff
@@ -1,97 +1,59 @@
...
+Weave GitOps’ [intuitive user interface](./getting-started/ui/) surfaces key information to help application operators easily discover and resolve issues—simplifying and scaling adoption of GitOps and continuous delivery. The UI provides a guided experience that helps users to easily discover the relationships between Flux objects and build understanding while providing insights into application deployments.
 
-import Link from '@docusaurus/Link';
+Today Weave GitOps defaults are Flux, Kustomize, Helm, SOPS, and Kubernetes Cluster API. If you use Flux already, then you can easily add Weave GitOps to create a platform management overlay.
 
-Need even more GitOps? <Link to="../intro-ee">**Weave GitOps Enterprise Edition (WGEE)**</Link> has all the features
-listed above, plus many more.
+:::tip Adopting GitOps can bring a number of key benefits—including faster and more frequent deployments, easy recovery from failures, and improved security and auditabiity. Check out our [GitOps for Absolute Beginners](https://go.weave.works/WebContent-EB-GitOps-for-Beginners.html) eBook and [Guide to GitOps](https://www.weave.works/technologies/gitops/) for more information.
+:::
```
Comment (Himangini) [inline website/docs/intro.mdx:19]:
```suggestion
:::tip 
Adopting GitOps can bring a number of key benefits—including faster and more frequent deployments, easy recovery from failures, and improved security and auditabiity. Check out our [GitOps for Absolute Beginners](https://go.weave.works/WebContent-EB-GitOps-for-Beginners.html) eBook and [Guide to GitOps](https://www.weave.works/technologies/gitops/) for more information.
:::
```

File: website/docs/intro.mdx [hunk 6]
```diff
@@ -1,97 +1,59 @@
...
-- :black_right_pointing_double_triangle_with_vertical_bar: **Progressive Delivery**
-- :point_up_2: **Self-Service Templates and Profiles**
-- :sparkling_heart: **Health Status and Compliance Dashboards**
-- :factory_worker::female-factory-worker: **Team Workspaces**
-- :compass: **Kubernetes Anywhere**
-- :bell: **Critical 24/7 Support**
-- :infinity: **CD Pipelines**
+This user guide provides content that will help you to [install](./installation/index.mdx) and [get started](./getting-started/intro.mdx) with our free and paid offerings:
+- **Weave GitOps Open Source**: a simple, open source developer platform for people who don't have Kubernetes expertise but who want cloud native applications. It includes the UI and many other features that take your team beyond a simple CI/CD system. Experience how easy it is to enable GitOps and run your apps in a cluster. 
+- **Weave GitOps Enterprise**: an [enterprise version](./next/intro-ee.mdx) that adds automation and 100% verifiable trust to existing developer platforms, enabling faster and more frequent deployments with guardrails and golden paths for every app team.
```
Comment (Himangini) [inline website/docs/intro.mdx:25]:
```suggestion
- **Weave GitOps Enterprise**: an [enterprise version](./getting-started/intro-ee.mdx) that adds automation and 100% verifiable trust to existing developer platforms, enabling faster and more frequent deployments with guardrails and golden paths for every app team.
```
the link is broken

File: website/docs/intro.mdx [hunk 7]
```diff
@@ -1,97 +1,59 @@
...
+- :mag: Uncover relationships between resources and quickly navigate between them.
+- :thinking_face: Understand how workloads are reconciled through a directional graph.
+- :goggles: View Kubernetes events relating to a given object to understand issues and changes.
+- :no_pedestrians: Secure access to the dashboard through the ability to integrate with an OIDC provider (such as Dex).
 
-- [GitOps for absolute beginners](https://go.weave.works/WebContent-EB-GitOps-for-Beginners.html) - eBook from Weaveworks
-- [Guide to GitOps](https://www.weave.works/technologies/gitops/) - from Weaveworks
-- [OpenGitOps](https://opengitops.dev/) - CNCF Sandbox project aiming to define a vendor-neutral, principle-led meaning of GitOps.
-- [gitops.tech](https://www.gitops.tech/) - supported by Innoq
+OK, time to [get started](./getting-started/intro/)!
```
Comment (Himangini) [inline website/docs/intro.mdx:56]:
```suggestion
OK, time to [get started](./getting-started/intro.mdx)!
```
broken link

File: website/docs/intro.mdx [hunk 8]
```diff
@@ -1,97 +1,59 @@
...
-  - :electric_plug: Compatible with any conformant [Kubernetes version](https://fluxcd.io/docs/installation/#prerequisites) and common ecosystem technologies such as Helm, Kustomize, RBAC, Prometheus, OPA, Kyverno, etc.
-  - :office: Multitenancy, multiple git repositories, multiple clusters
-  - :exclamation: Alerts and notifications
+Weave GitOps improves developer experience—simplifying the complexities and cognitive load of deploying and managing cloud native apps on Kubernetes so that teams can go faster. It’s a powerful extension of Flux, a leading GitOps engine and [Cloud Native Computing Foundation project](https://www.cncf.io/projects/). [Weaveworks](https://weave.works) are the creators of [Flux][flux].
 
-## Weave GitOps Enterprise
+Weave GitOps’ [intuitive user interface](./getting-started/ui/) surfaces key information to help application operators easily discover and resolve issues—simplifying and scaling adoption of GitOps and continuous delivery. The UI provides a guided experience that helps users to easily discover the relationships between Flux objects and build understanding while providing insights into application deployments.
```
Comment (Himangini) [inline website/docs/intro.mdx:14]:
```suggestion
Weave GitOps’ [intuitive user interface](./getting-started/ui.mdx) surfaces key information to help application operators easily discover and resolve issues—simplifying and scaling adoption of GitOps and continuous delivery. The UI provides a guided experience that helps users to easily discover the relationships between Flux objects and build understanding while providing insights into application deployments.
```

broken link, it should be [intuitive user interface](./getting-started/ui.mdx)

**Review by Himangini — COMMENTED** [review]
small nits 😅 

tip 💡 if you put text next to admonitions head it becomes a title https://docusaurus.io/docs/markdown-features/admonitions#specifying-title

File: website/docs/installation/weave-gitops.mdx (hunk 4 above)
Comment (LappleApple) [reply website/docs/installation/weave-gitops.mdx]: @Himangini Thank you, I've adjusted

File: website/docs/intro.mdx [hunk 9]
```diff
@@ -1,97 +1,60 @@
...
-- :black_right_pointing_double_triangle_with_vertical_bar: **Progressive Delivery**
-- :point_up_2: **Self-Service Templates and Profiles**
-- :sparkling_heart: **Health Status and Compliance Dashboards**
-- :factory_worker::female-factory-worker: **Team Workspaces**
-- :compass: **Kubernetes Anywhere**
-- :bell: **Critical 24/7 Support**
-- :infinity: **CD Pipelines**
+This user guide provides content that will help you to [install](./installation/index.mdx) and [get started](./getting-started/intro.mdx) with our free and paid offerings:
+- **Weave GitOps Open Source**: a simple, open source developer platform for people who don't have Kubernetes expertise but who want cloud native applications. It includes the UI and many other features that take your team beyond a simple CI/CD system. Experience how easy it is to enable GitOps and run your apps in a cluster. 
+- **Weave GitOps Enterprise**: an [enterprise version](./getting-started/intro-ee.mdx) that adds automation and 100% verifiable trust to existing developer platforms, enabling faster and more frequent deployments with guardrails and golden paths for every app team.
```
Comment (Himangini) [inline website/docs/intro.mdx:26]:
```suggestion
- **Weave GitOps Enterprise**: an [enterprise version](./intro-ee.mdx) that adds automation and 100% verifiable trust to existing developer platforms, enabling faster and more frequent deployments with guardrails and golden paths for every app team.
```
small change for the link , it should work now ✨

**Comment by yiannistri** [comment]
Reviewing on https://staging.docs.gitops.weave.works/refactor-docs

**Review by yiannistri — COMMENTED** [review]
Nice work, some suggestions on formatting but other than that, the content is clear and helpful.

File: website/docs/getting-started/ui.mdx [hunk 10]
```diff
@@ -1,112 +1,141 @@
...
+To check out Weave GitOps Enterprise's UI, which provides an even richer user experience, please contact sales@weave.works.
+:::
+
+## Overview
+
+A quick preview of what the Weave GitOps Open Source UI provides:
+* an **Applications view** that shows summary information from `Kustomization` and `HelmRelease` objects so that you can quickly understand the state of your deployments across a cluster.
+* a **Sources view** that shows summary information from gitrepository, helmrepository and bucket objects and tells you the current status of resources that are synchronizing content from where you’ve declared the desired state of your system—for example, Git repositories.
+* a **Flux Runtime view** that provides the status of the GitOps engine that continuously reconciles your desired and live state. It shows your installed [GitOps Toolkit Controllers](https://fluxcd.io/flux/components/) and version.
+* multiple views for debugging
```
[the remark this answers is not in the document]
Comment (yiannistri) [reply website/docs/getting-started/ui.mdx:20]: At some point (in a separate PR) we should update the screenshots to match latest version. The existing screenshots are a bit out of date.

File: website/docs/getting-started/ui.mdx [hunk 11]
```diff
@@ -1,112 +1,141 @@
 ---
-title: 1. Explore the GitOps UI
+title: 1. Explore the Weave GitOps Open Source UI
 hide_title: true
 ---
 
-# Part 1: Explore the GitOps UI
+# Step 2: Explore the Weave GitOps Open Source UI
```
[the remark this answers is not in the document]
Comment (LappleApple) [reply website/docs/getting-started/ui.mdx:6]: @yiannistri Understood, I likely made this change in anticipation of https://github.com/weaveworks/weave-gitops/issues/3739 changes. I will change back for now until we implement the new set of changes.

File: website/docs/getting-started/ui.mdx (hunk 10 above)
[the remark this answers is not in the document]
Comment (LappleApple) [reply website/docs/getting-started/ui.mdx:20]: @yiannistri Yeah, I was planning to get to Image Automation and Notifications in another PR. Late yesterday I found a source I can pull docs from. :)

File: website/docs/getting-started/ui.mdx [hunk 12]
```diff
@@ -1,112 +1,141 @@
...
+Go back to the Details tab, and click `GitRepository/flux-system` from the summary at the top of the page.
 
-![Flux Runtime view showing the various GitOps Toolkit controllers](/img/dashboard-flux-runtime-crd.png)
+![Source detail view showing details for an object](/img/dashboard-source-flux.png)
 
-### Exploring the flux-system deployment
+As with an Application detail view, you can see key information about how the resource is defined.
 
-Let's explore the `flux-system` kustomization. Navigate back to the `Applications` view and click on the `flux-system` object.
+### The Flux Runtime view
```
[the remark this answers is not in the document]
Comment (LappleApple) [reply website/docs/getting-started/ui.mdx:115]: Thanks for this catch

**Review by yiannistri — APPROVED** [review]
Looks great ✨ Let's change "Part" to "Step" for steps 0 and 2 in a follow-up PR

**Comment by Himangini** [comment]
I was about to comment but this got merged 😅 
The following few places have formatting issues
Screenshot 2023-06-01 at 14 54 20
Screenshot 2023-06-01 at 14 56 22

**Comment by Himangini** [comment]
Also, these changes are not part of the latest versions 
Screenshot 2023-06-01 at 15 04 20

**Comment by LappleApple** [comment]
Thanks @Himangini  -- I'm going to open another PR tomorrow to continue with this work and will fold in these issues into that PR.

**Pull request merged**